src/lib/SCREEN/screen.h

```diff
@@ -0,0 +1,222 @@
+#pragma once
+
+#include "targets.h"
+#include "axis_logo.h"
```

Comment:
Screen/displays hardware will need to be generic and non target specific.

Any logos will need to be wrapped in a target specific define.

Reply:
Just move logo to target define header and keep others icon define, is it OK?

Reply:
Personally, I'd have a `logos.h`, and in that file wrap the axis logo in an `ifdef`.
I think this is what @JyeSmith is meaning.

Reply:
@pkendall64 that sounds good.

Reply:
update it at d617ecf